pkg/controllers/metrics/nodepool/controller.go

```diff
@@ -41,14 +41,13 @@ import (
 const (
 	resourceTypeLabel = "resource_type"
 	nodePoolNameLabel = "nodepool"
-	nodePoolSubsystem = "nodepool"
 )
 
 var (
 	limitGaugeVec = prometheus.NewGaugeVec(
 		prometheus.GaugeOpts{
 			Namespace: metrics.Namespace,
-			Subsystem: nodePoolSubsystem,
+			Subsystem: metrics.NodePoolSubsystem,
 			Name:      "limit",
 			Help:      "The nodepool limits are the limits specified on the nodepool that restrict the quantity of resources provisioned. Labeled by nodepool name and resource type.",
```

Comment:
nit: think it's worth changing the help caption
```suggestion
			Help:      "Limits specified on the nodepool that restrict the quantity of resources provisioned. Labeled by nodepool name and resource type.",
```